config/features/flags.go

```diff
@@ -176,6 +176,10 @@ var (
 		Name:  "enable-experimental-attestation-pool",
 		Usage: "Enables an experimental attestation pool design.",
 	}
+	EnableStateDiff = &cli.BoolFlag{
```

Comment:
This one does not need to be exported, it may confuse someone to have the same name as the Flag one that does need to be exported.